docs/reference/method/MongoDBClient__construct.txt

```diff
@@ -19,7 +19,7 @@ Definition
 
    .. code-block:: php
 
-      function __construct($uri = 'mongodb://127.0.0.1/', array $uriOptions = [], array $driverOptions = [])
+      function __construct(?string $uri = null, array $uriOptions = [], array $driverOptions = [])
```

Comment:
The `$uri` docs on https://www.mongodb.com/docs/php-library/current/reference/method/MongoDBClient__construct/ already state:

> Defaults to "mongodb://127.0.0.1:27017" if unspecified.

So there's no need to change anything else on this page. I also don't think there's any need to document the constant you introduced.